README.md

```diff
@@ -72,14 +72,23 @@ php artisan db:seed --class=UsersTableSeeder
 ```
 #### Indexing Solr Cores
 (The data import functionality is deprecated and scheduled to be removed in Solr 9.)
-2. Use the Solr web app to import data from MySQL to index each Jitterbug core. 
+2. You must create empty `dataimport.properties` files in the following Solr config directories for each core:
+```bash
+    solrconfig/jitterbug-instances/conf
+    solrconfig/jitterbug-items/conf
+    solrconfig/jitterbug-transfers/conf
+```
+3. Use the Solr web app to import data from MySQL to index each Jitterbug core. 
 	1. Point your favorite web browser to http://homestead.test:8983/solr
 	2. Use the Solr "Core Selector" menu to select the jitterbug-items core.
 	3. Click the Dataimport button under the Core Selector menu.
 	4. Underneath the Execute button, check the Auto-Refresh Status checkbox.
 	5. Click Execute.
 	6. When jitterbug-items is finished indexing, repeat these steps for each core.
-
+- If the browser cannot connect, open virtualbox and click on your VM. 
+  Then click the settings icon and click on Network, and expand the Advanced dropdown.
+  Click on port forwarding and add 8983 to the list by clicking on the green plus in the top right.
```

Comment:
Is this an issue with Homestead? Is that port set to be forwarded somewhere?

Reply:
I'm not sure what happened, but I think with the last upgrade or so accessing the solr admin page stopped working. It wouldn't connect, and after troubleshootng with @bbpennel he suggested adding the port manually in virtualbox and it worked. I realized that I didn't have it written down anywhere so I thought i should add it.

Reply:
OK, that sounds fine, thanks.